.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us fix the issue
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## The problem
+
+Briefly describe the issue you are experiencing (or the feature you want to see added to Percy). Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to https://github.com/percy/cli/discussions/new
+
+## Reach out to Percy support
+
+If you’re having issues that _aren’t SDK bugs_, it would be best for you to reach out to support instead: support@percy.io or https://www.browserstack.com/contact#technical-support
+
+## Environment
+
+- Node version
+- What version of  `@percy/cli`
+- What version of Percy SDK you’re using
+- If needed, a build or snapshot ID
+
+## Details
+
+If necessary, describe the problem you have been experiencing in more detail.
+
+## Debug logs
+
+If you are reporting a bug, _always_ include logs! Run a Percy build with `--verbose` to get the full debug logs. For example, `percy exec --verbose -- [test command]`.  Please include the full complete test output. 
```

Comment:
Should we mention `PERCY_LOGLEVEL=debug`? In some CI envs, it might be easier to add an env var from the UI rather than alter the command used to run tests.